core/templates/dev/head/filters.js

```diff
@@ -342,6 +342,29 @@ oppia.filter('summarizeNonnegativeNumber', [function() {
   };
 }]);
 
+// Note that this filter removes additional new lines or <p><br></p> tags
+// at the end of the string.
+oppia.filter('removeExtraLines', [function() {
+  return function(string) {
+    if (!angular.isString(string)) {
+      return string;
+    }
+    while (1) {
+      var lastIndex = string.length;
+      var BLANK_LINES_TEXT = '<p><br></p>';
+      if (string.substring(
+        lastIndex - BLANK_LINES_TEXT.length, lastIndex
+      ) === BLANK_LINES_TEXT) {
+        string = string.substring(0, lastIndex - BLANK_LINES_TEXT.length);
+      }
+      else {
```

Comment:
If I had to guess, I think this is causing the linting error, try combining the `else {` with the previous line so it looks like `} else {`.

It's strange that the linter doesn't alert you about this error locally, I would investigate why that's happening also. Are you running the linter before or after you commit?

Reply:
@AllanYangZhou, linter automatically runs before `git push` via `pre_push_hook.py`.
Also, I found this in `pre_push_hook.py`:
>This hook works only on Unix like systems as of now.
On Vagrant under Windows it will still copy the hook to the .git/hooks dir
but it will have no effect.

Can this be the reason?

Reply:
@AllanYangZhou and @DubeySandeep: I think the fork is not updated.  
Thanks!